xml/System.Net.Http/HttpCompletionOption.xml

```diff
@@ -36,7 +36,27 @@
   </Base>
   <Docs>
     <summary>Indicates if <see cref="T:System.Net.Http.HttpClient" /> operations should be considered completed either as soon as a response is available, or after reading the entire response message including the content.</summary>
-    <remarks>To be added.</remarks>
+    <remarks>
```

Comment:
I couldn't find a way to preview these changes, all the tooling seems to work only for markdown files, not these XMLs: https://review.learn.microsoft.com/en-us/help/get-started/?branch=main&tabs=learn. So I don't really know what these changes look like.

Reply:
You can't preview changes. Repository maintainers can preview using their internal tool. Currently your PR has build warnings due to using `<see>` tags in markdown, so it will likely render incorrectly in preview. You need to fix warnings by using xrefs instead of `<see>` to reference APIs, like: `<xref:System.Net.Http.HttpClient>`. Look existing remarks for examples.

Reply:
I've replaced the `<see>` tags with `<xref>`, hopefully it renders correctly.